www/grpc/proto/wallet.proto

```diff
@@ -40,6 +40,40 @@ service Wallet {
   // public key.
   rpc GetValidatorAddress(GetValidatorAddressRequest)
       returns (GetValidatorAddressResponse);
+
+  // GetNewAddress generates a new address for the specified wallet.
+  rpc GetNewAddress(GetNewAddressRequest) returns (GetNewAddressResponse);
+}
+
+// Enum for the address type.
+enum AddressType {
+  ADDRESS_TYPE_ADDRESS_TYPE_TREASURY = 0;
```

Comment:
I think it can be `ADDRESS_TYPE_TREASURY` or even `TREASURY`. Same for other values